idaes/core/base/flowsheet_model.py

```diff
@@ -342,35 +367,24 @@ def _setup_dynamics(self):
                     "{} was set as a dynamic flowsheet, but time domain "
                     "provided was not a ContinuousSet.".format(self.name)
                 )
+            
+            if self.config.dynamic is False and not isinstance(
```

Comment:
1. Is this check actually necessary? This would appear to preclude putting steady-state flowsheets inside dynamic flowsheets.
2. `if self.config.dynamic is False` can be written as `if not self.config.dynamic`

Reply:
That's exactly what it was checking for. Since we don't want to allow adding a nested dynamic flowsheet inside a parent steady-state flowsheet, I will remove this check from this PR.